website/content/v1.7/talos-guides/install/cloud-platforms/aws.md

```diff
@@ -7,52 +7,102 @@ aliases:
 
 ## Creating a Cluster via the AWS CLI
 
-In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
-We assume an existing VPC, and some familiarity with AWS.
+In this guide we will create an HA Kubernetes cluster with 3 control plane nodes across 3 availability zones.
+You should have an existing AWS account and have the AWS CLI installed and configured.
 If you need more information on AWS specifics, please see the [official AWS documentation](https://docs.aws.amazon.com).
 
-### Set the needed info
-
-Change to your desired region:
+To install the dependencies for this tutorial you can use homebrew on macOS or Linux:
 
 ```bash
-REGION="us-west-2"
-aws ec2 describe-vpcs --region $REGION
+brew install siderolabs/tap/talosctl kubectl jq ipcalc curl xz
+```
+
+If you would like to create infrastructure via `terraform` or `opentofu` please see the exmaple in the [contrib repository](https://github.com/siderolabs/contrib/tree/main/examples/terraform/aws).
+
+> Note: this guide is not a production set up and steps were tested in `bash` and `zsh` shells.
+
+### Create AWS Resources
 
-VPC="(the VpcId from the above command)"
+We will be creating a control plane with 3 Ec2 instances spread across 3 availability zones.
+It is recommended to not use the default VPC so we will create a new one for this tutorial.
+
+Change to your desired region and CIDR block and create a VPC:
+
+> Make sure your subnet does not overlap with `10.244.0.0/16` or `10.96.0.0/12` the [default pod and services subnets in Kubernetes]({{% ref "/v1.7/introduction/troubleshooting.md#conflict-on-kubernetes-and-host-subnets" %}}).
```

Comment:
we should use `relref`, not `ref`, and without the version, to keep docs same across versions